cluster/src/main/scala/org/apache/spark/sql/SnappySessionFactory.scala

```diff
@@ -98,14 +76,37 @@ trait SnappySQLJob extends SparkJobBase {
     }
   }
 
-  private def cleanJobConfig(c: Config): Config = {
+  def cleanJobConfig(c: Config): Config = {
     // TODO Remove snappydata properties path when available
     var sJobConfig = c.withoutPath(Constant.STORE_PROPERTY_PREFIX + com.pivotal.gemfirexd
       .Attribute.USERNAME_ATTR)
     sJobConfig = sJobConfig.withoutPath(Constant.STORE_PROPERTY_PREFIX + com.pivotal
       .gemfirexd.Attribute.PASSWORD_ATTR)
     sJobConfig
   }
+}
+
+
+trait SnappySQLJob extends SparkJobBase {
+  type C = Any
+
+  final override def validate(sc: C, config: Config): SparkJobValidation = {
+    val parentLoader = org.apache.spark.util.Utils.getContextOrSparkClassLoader
+    val currentLoader = SnappyUtils.getSnappyStoreContextLoader(parentLoader)
+    Thread.currentThread().setContextClassLoader(currentLoader)
+    SnappyJobValidate.validate(isValidJob(sc.asInstanceOf[SnappySession], SnappySessionFactory
+        .cleanJobConfig(config)))
+  }
```

Comment:
For a single line change why it is showing such a big diff. I dont see any change in indentation too. Can you try to restore file and later do you 2-3 lines of changes. You can restore file as:
git checkout master cluster/src/main/scala/org/apache/spark/sql/SnappySessionFactory.scala